integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/BenchUtils.scala

```diff
@@ -520,12 +520,7 @@ object BenchUtils {
 
```

Comment:
2021 copyrights need to be added to this file.

Reply:
added.